sys/fido2/ctap/ctap_utils.c

```diff
@@ -79,39 +77,43 @@ static void _gpio_cb(void *arg)
     _user_present = true;
 }
 
-void fido2_ctap_utils_led_animation(void)
+void fido2_ctap_utils_wait_for_user_presence(void)
 {
     uint32_t start = ztimer_now(ZTIMER_MSEC);
     uint32_t diff = 0;
     uint32_t delay = 500;
 
     while (!_user_present && diff < CTAP_UP_TIMEOUT) {
-#ifdef LED0_TOGGLE
+#if !IS_ACTIVE(CONFIG_FIDO2_CTAP_DISABLE_LED)
+    #ifdef led0_toggle
         LED0_TOGGLE;
-#endif
-#ifdef LED1_TOGGLE
+    #endif
+    #ifdef LED1_TOGGLE
         LED1_TOGGLE;
-#endif
-#ifdef LED3_TOGGLE
-        LED3_TOGGLE;
-#endif
-#ifdef LED2_TOGGLE
+    #endif
+    #ifdef LED2_TOGGLE
         LED2_TOGGLE;
+    #endif
+    #ifdef LED3_TOGGLE
+        LED3_TOGGLE;
+    #endif
```

Comment:
```suggestion
        LED3_TOGGLE;
```

and including "led.h" should be enough. Same for all the other places.

Reply:
Other examples I have found in the riot code ([1](https://github.com/Ollrogge/RIOT/blob/e4b5bf50111bee900ccd2f6a59c280f70e7c1653/examples/blinky/main.c#L54), [2](https://github.com/Ollrogge/RIOT/blob/e4b5bf50111bee900ccd2f6a59c280f70e7c1653/tests/leds/main.c#L58)) use `ifdef` guards.